api/src/controllers/relPersonPlace.js

```diff
@@ -63,7 +63,21 @@ router.delete(
   passport.authenticate("user", { session: false }),
   catchErrors(async (req, res) => {
     const { _id } = req.params;
-    await RelPersonPlace.destroy({ where: { _id } });
+    try {
+      await RelPersonPlace.destroy({ where: { _id } });
+    } catch (e) {
+      const query = {
+        where: {
+          _id: req.params._id,
+          organisation: req.user.organisation,
+        },
+      };
+
+      const relPersonPlace = await RelPersonPlace.findOne(query);
+      relPersonPlace.set({ user: req.user._id });
+      await relPersonPlace.save();
+      await RelPersonPlace.destroy({ where: { _id } });
+    }
```

Comment:
Pourquoi pas plutôt null ? On comprends peut-être mieux l'intention (attention code non testé)

```suggestion
    try {
      await RelPersonPlace.destroy({ where: { _id } });
    } catch (e) {
      const relPersonPlace = await RelPersonPlace.findOne({ where: { _id } });
      relPersonPlace.set({ user: null });
      await relPersonPlace.save();
      await RelPersonPlace.destroy({ where: { _id } });
    }
```

Side note: j'ai pas compris pourquoi on ajoutait l'organisation dans le where dans ce cas uniquement...